docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -777,10 +776,9 @@ to verify the syncer pod is running.
 ```
 
 ## Creating a Workload Management Space
```

Comment:
"workload management space" is neither the old term nor the new term. Pick one or the other.

Reply:
Done in latest push. Using the old term here for consistency with the context.

Reply:
OK, but that means that we need another PR to update to the new terminology.

Reply:
I assumed that.

Reply:
OK